src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -42,54 +70,51 @@ export class BidResult extends React.Component<BidResultProps> {
 
   exitBidFlow = async () => {
     await SwitchBoard.dismissModalViewController(this)
-    if (this.props.status === "LIVE_BIDDING_STARTED") {
+
+    if (this.props.bidderPositionResult.status === "LIVE_BIDDING_STARTED") {
       SwitchBoard.presentModalViewController(this, `/auction/${this.props.sale_artwork.sale.id}`)
     }
   }
 
   render() {
-    const { live_start_at, end_at } = this.props.sale_artwork.sale
-
-    // non-live sale doesn't have live_start_at so bidding is open until end time
-    if (this.props.winning) {
-      return (
-        <BiddingThemeProvider>
-          <Container mt={6}>
-            <View>
-              <Flex alignItems="center">
-                <Icon20 source={require("../../../../../images/circle-check-green.png")} />
-                <Title m={4}>You're the highest bidder</Title>
-                <Timer liveStartsAt={live_start_at} endsAt={end_at} />
-              </Flex>
-            </View>
+    const { sale_artwork, bidderPositionResult } = this.props
+    const { live_start_at, end_at } = sale_artwork.sale
+    const { status, message_header, message_description_md, position } = bidderPositionResult
+
+    const nextBid = ((position && position.suggested_next_bid) || sale_artwork.minimum_next_bid || ({} as any)).display
+
+    return (
+      <BiddingThemeProvider>
+        <Container mt={6}>
+          <View>
+            <Flex alignItems="center">
+              <Icon20 source={Icons[status] || require("../../../../../images/circle-x-red.png")} />
+
+              <Title m={4}>
+                {status === "PENDING" ? "Bid Processing" : message_header || "You're the highest bidder"}
```

Comment:
:D we could knock out a QA ticket by renaming this to `Bid processing` (since they should all be sentence case). Also it looks like the apostrophe in `You're` should be a smart quote.

Reply:
fixed :+1: